sdk/subscription/ci.yml

```diff
@@ -1,32 +0,0 @@
-# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
We shouldn't be deleting `ci.yml`. This is your build/release pipeline definition essentially.